### ISSUE TITLE
missing info.apiURL in pom.xml ?

### ISSUE THREAD

**Issue by patriknw** [akka/akka-http#449]
When running `sbt eclipse` in akka-http there are many of the following warnings. Is there something missing in the build?

```
[warn] 	:: com.typesafe.akka#akka-http_2.11;3.0.0-RC1: java.text.ParseException: inconsistent module descriptor file found in 'https://repo1.maven.org/maven2/com/typesafe/akka/akka-http_2.11/3.0.0-RC1/akka-http_2.11-3.0.0-RC1.pom': bad info.apiURL found in https://repo1.maven.org/maven2/com/typesafe/akka/akka-http_2.11/3.0.0-RC1/akka-http_2.11-3.0.0-RC1.pom: expected='http://doc.akka.io/api/akka-http/3.0.0-RC1/' found='null';
```

**Comment by jonas** [comment]
`apiURL` was configured in ab9fe6483f8e689ceca678bb11166ed239a0443f so hasn't yet been included in any RC.

**Comment by patriknw** [comment]
ok, good, why is the version number in master `3.0.0-RC1`? that should always be bumped to a -SNAPSHOT after releasing (10.0.0-SNAPSHOT)

**Comment by jonas** [comment]
I am not sure, but this might be fixed by #388.
WDYT @jrudolph?

**Comment by patriknw** [comment]
yes, it will
